SwiftUIDZ/SwiftUIDZ/ClimateControlView/ClimateControlView.swift

```diff
@@ -0,0 +1,350 @@
+                        Image(systemName: climateControlViewModel.settingsClimate[index].settingsImageName)
+                            .foregroundColor(climateControlViewModel.settingsClimate[index].isAction ? Color("GradientBottom") : .white)
+                    }
+                }
+
+              
+                
+                LockerSlider(value: Binding(get: {
+                    if climateControlViewModel.settingsClimate[index].isAction {
+                     return   climateControlViewModel.climateSliderValues[index]
+                    } else {
+                     return   0.0
+                    }
+                }, set: { newValue in
+                    if climateControlViewModel.settingsClimate[index].isAction {
+                        climateControlViewModel.climateSliderValues[index] = newValue
+                    } else {
+                        climateControlViewModel.climateSliderValues[index] = 0.0
+                    }
+                }), in: 0...15, step: 1)
+                    .frame(width: 230, height: 50)
+            }
+        }
+        .padding(.vertical, 12)
+    }
+    
+    var body: some View {
+       // NavigationView {
+            ZStack {
+                BackgroundStackView(colors: climateControlViewModel.gradientColors) {
+                VStack(spacing: 10) {
+                    HStack(spacing: 55) {
+                        backButtonView
+                        Text("CLIMATE")
+                            .font(.system(size: 20))
+                            .bold()
+                            .font(.title2)
+                        settingsButtonView
+                    }
+                    .padding(.top, 20)
+                    temperatureIndicatorView
+                    DisclosureGroup {
+                        ForEach(0 ..< 4) { index in
+                            makeClimateGroupView(index: index)
+                        }
+                    } label: {
+                        Text("Настройки")
+                    }
+                    .foregroundColor(.white)
+                    .tint(.white)
+                  //  .frame(width: 450)
+                    .padding(.horizontal, 10)
+                    bottomSheetView
+                    Spacer()
+                }
+                
+            }
+           
+        }
+            .navigationBarBackButtonHidden()
+        
+        
+    }
+    
+    private var toolbarTitleTextView: some View {
+        Text("CLIMATE")
+    }
+    
+    private var backButtonView: some View {
+        СonvexCircleButtonView(actionHandler: {
+            presentation.wrappedValue.dismiss()
+        }, iconName: "chevron").tint(.white)
+    }
+    
+    private var settingsButtonView: some View {
+        СonvexCircleButtonView(actionHandler: {
+            print("")
+        }, iconName: "settings").tint(.white)
+    }
+        
+//    private var gradientView: GradientsBackgroundStackView {
```

Comment:
```suggestion
//    private var gradientView: GradientsBackgroundStackView {
```
убери весь закоменченый код